src/tlitest/tlitest-run

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+cd $(dirname $0)
+echo "RUN_PATH=$PWD"
+
+TEST_DIRS="
+../../lib
+/usr/local/lib
+/lib
+"
+
+for TEST_DIR in $TEST_DIRS; do
+    TEST_FILE=$(find $TEST_DIR -name test_tlog.py|head -1)
+    if [ -n "$TEST_FILE" ]; then
+        break
+    fi
+done
+
+if [ -z "$TEST_FILE" ]; then
+    echo "Cannot find test_tlog.py pytest file"
+    exit 1
+fi
+
+echo "Found $TEST_FILE...changing directory to run pytest"
+cd $(dirname $TEST_FILE)
+pytest-3 -v test_tlog.py $*
```

Comment:
Another thing: using `$*` would break as soon as you have whitespace in your arguments, which is a nasty surprise. So, in general use "$@", which will reproduce the arguments faithfully.

Reply:
Forgot about $@.  I will switch as suggested.  Thanks